core/domain/classifier_services_test.py

```diff
@@ -173,3 +173,78 @@ def test_update_of_classifiers(self):
         self.assertEqual(classifier.exp_id, exp_id)
         self.assertEqual(classifier.state_name, test_state_name)
         self.assertEqual(classifier.id, classifier_id)
+
+    def test_retrieval_of_classifier_training_jobs(self):
+        """Test the get_classifier_training_job_by_id method."""
+
+        with self.assertRaisesRegexp(Exception, (
+            "Entity for class ClassifierTrainingJobModel with id fake_id not found")):
+            classifier_services.get_classifier_training_job_by_id('fake_id')
+
+        exp_id = u'1'
+        state_name = 'Home'
+        job_id = classifier_models.ClassifierTrainingJobModel.create(
+            feconf.INTERACTION_CLASSIFIER_MAPPING['TextInput']['algorithm_id'],
+            exp_id, 1, [], state_name)
+        classifier_training_job = (
+            classifier_services.get_classifier_training_job_by_id(job_id))
+        self.assertEqual(classifier_training_job.exp_id, exp_id)
+        self.assertEqual(classifier_training_job.state_name, state_name)
+        self.assertEqual(classifier_training_job.status, 'NEW')
+
+    def test_deletion_of_classifier_training_jobs(self):
+        """Test the delete_classifier_training_job method."""
+
+        with self.assertRaisesRegexp(Exception, (
+            "Entity for class ClassifierTrainingJobModel with id fake_id not found")):
```

Comment:
@seanlip I have a small Linting issue here. The line comes out to be too long. But, I can't wrap it the next line since that introduces a new-line character in between the string and then, the exception raised does not equal the string we give it. How can I fix this?